### ISSUE TITLE
New version detection - error

### ISSUE THREAD

**Issue by svatas** [otros-systems/otroslogviewer#546]
When I download latest version 1.4.12 and start it, I got immediatelly a message, that there is a new version 1.4.9 available, what is wrong.
![image](https://user-images.githubusercontent.com/2070628/45416515-64eb2000-b680-11e8-9aae-cdbf5a76af5e.png)

The latest is 1.4.12. 

This bug works also vice versa. If I install 1.4.9, there is a message that this is latest version.
![image](https://user-images.githubusercontent.com/2070628/45416477-56046d80-b680-11e8-822d-a09fe09049a1.png)

Same issue with 1.4.10 and 1.4.11, all are linking to 1.4.9.

**Comment by Simulant87** [comment]
The problem is located here: `CheckForNewVersionAbstract#done()` line 68 `if (current.compareTo(running) > 0)` the version Strings are compared with the default `String#compareTo()` therefore `9 > 10`. The solution would be to parse the three parts of the version String into numbers and compare the integers.

**Comment by svennissel** [comment]
I think the problem is the not responding URL:
http://otroslogviewer.appspot.com/services/currentVersion?runningVersion=1.4.13&java.version=1.8.0_322&os.name=Linux&vm.vendor=Amazon.com+Inc.&uuid=1187f187-ec57-4072-873f-1dec947c79b3

The server Response:

```
Error: Server Error
The server encountered an error and could not complete your request.

Please try again in 30 seconds.
```

**Comment by svennissel** [comment]
I think it must be rewritten. See #534

**Comment by svennissel** [comment]
Fixed with #602
